### PR TITLE
S3 setup

### PR DESCRIPTION
**Pull request by singonwan**
Using axios for the moment, will change to http hooks once we find a way

### CODE REVIEW

File: server/controllers/users-controller.js [hunk 1]
```diff
@@ -143,13 +143,18 @@ const login = async (req, res, next) => {
...
 const getAllReceipt = async (req, res, next) => {
     const userId = req.userData.userId;
+    const { duration } = req.body;
 
     try {
-        const receipts = await Receipt.find({ user: userId });
+        const receipts = await Receipt.find({ user: userId }).sort({
+            date: -1,
+        });
+
```
Comment (singonwan) [inline server/controllers/users-controller.js:157]: dont worry about this, this is for my next task

File: server/models/receipt.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ const Receipt = mongoose.model(
         },
         date: {
             type: Date,
+            default: Date.now,
```
Comment (singonwan) [inline server/models/receipt.js:20]: this is also for the next task, but this is added so all receipts added will default have the date that they create the receipt.

File: server/routes/s3-routes.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+const s3_controller = require('../controllers/s3-controller');
+const checkAuth = require('../middleware/check-auth');
+
+// router.use(checkAuth);
```
Comment (singonwan) [inline server/routes/s3-routes.js:7]: will add this back on as soon as the button gets integrated to the receipt creation page

**Review by varunrampal — APPROVED** [review]
looks good, go with it.

**Pull request merged**